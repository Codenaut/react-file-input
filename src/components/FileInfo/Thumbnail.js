import React from 'react';
import Proptypes from 'prop-types';

import '../../styles/FileInfo.scss';

const Thumbnail = ({ children }) => (
  <div className="brainhub-file-info__thumbnail">
    {children}
  </div>
);

Thumbnail.propTypes = {
  children: Proptypes.node.isRequired,
};

export default Thumbnail;
