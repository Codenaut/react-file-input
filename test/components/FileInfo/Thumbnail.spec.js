import React from 'react';
import renderer from 'react-test-renderer';

import Thumbnail from '../../../src/components/FileInfo/Thumbnail';

describe('components', () => {
  describe('FileInfo', () => {
    describe('Thumbnail', () => {
      it('should match exact snapshot', () => {
        const tree = renderer.create(
          <div>
            <Thumbnail>
              <img src="imageSource"/>
            </Thumbnail>
          </div>
        ).toJSON();

        expect(tree).toMatchSnapshot();
      });
    });
  });
});
