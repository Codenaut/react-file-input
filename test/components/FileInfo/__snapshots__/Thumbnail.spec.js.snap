// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components FileInfo Thumbnail should match exact snapshot 1`] = `
<div>
  <div
    className="brainhub-file-info__thumbnail"
  >
    <img
      src="imageSource"
    />
  </div>
</div>
`;
